pkg/initializr/initializr.go

```diff
@@ -0,0 +1,166 @@
+package initializr
+
+import (
+	"archive/zip"
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// InitializrRequest contains options supported to generate a project using https://github.com/spring-io/initializr
+type InitializrRequest struct {
+	Dependencies string
+	GroupID      string
+	ArtifactID   string
+	Path         string
```

Comment:
feels like this struct should just have the fields needed for the request to initializr -- `Path` doesn't appear to be sent as part of the request. maybe `CreateNewProject` should take the path as an arg

Reply:
There is the alternative approach in https://github.com/markpollack/lift/tree/GH-5c (no global variables).  If we start to separate this out (which does express the intent better) in the current approach we will end up with more global variables.

Reply:
:+1: to the alt